tests/snuba/test_util.py

```diff
@@ -42,3 +44,29 @@ def test_referenced_columns(self):
             ],
         ]
         assert set(snuba.all_referenced_columns(conditions)) == set(['a', 'b', 'c', 'd'])
+
+    def test_shrink_timeframe(self):
+        now = datetime.now()
+        year_ago = now - timedelta(days=365)
+        year_ahead = now + timedelta(days=365)
+
+        issues = None
+        assert snuba.shrink_time_window(issues, year_ago, year_ahead) == (year_ago, year_ahead)
+
+        issues = []
+        assert snuba.shrink_time_window(issues, year_ago, year_ahead) == (year_ago, year_ahead)
+
+        group1 = self.create_group()
```

Comment:
Do we want to be explicit about what the first_seen and last_seen of this group is? or is it enough to just assume that they will be after year_ago and before year_ahead?

Reply:
Oops I meant to do that, good catch.